vllm/worker/worker_base.py

```diff
@@ -115,9 +117,22 @@ def update_environment_variables(self, envs: Dict[str, str]) -> None:
 
     def init_worker(self, *args, **kwargs):
         """
-        Actual initialization of the worker class.
+        Actual initialization of the worker class, and set up
+         logging if required.
         Arguments are passed to the worker class constructor.
         """
+        if int(os.getenv("VLLM_TRACE_FUNCTION", "0")):
+            from vllm.logger import enable_trace_function_call
+            from vllm.utils import get_vllm_instance_id
```

Comment:
global import is fine.

Reply:
fixed in [6e2609c](https://github.com/vllm-project/vllm/pull/4079/commits/6e2609c20fc185d60b521eb85aba4c0f90e6cc3a)